inst/WORDLIST

```diff
@@ -35,6 +35,7 @@ SUPPTR
 SV
 TAs
 Thampi
+USUBJID
```

Comment:
This can be removed if backticks are used in `R/data.R`.

Reply:
Updated as suggested.

Reply:
@bundfussr - After removing the USUBJID from the wordlist the following check is failing. Please suggest.
![image](https://github.com/pharmaverse/pharmaversesdtm/assets/90273880/79d1d678-febb-4925-95e6-86939ad95946)

Reply:
@PoojaKumari05 , there was a line missing in the `DESCRIPTION` file. Therefore then man files were not created correctly. I have fixed it.